tests/test-infra/find_cluster.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
```

Comment:
I usually use set -ue to enable the following:
-u Attempt to use undefined variable outputs error message, and forces an exit
-e Abort script at first error, when a command exits with non-zero status (except in until or while loops, if-tests, list constructs)





Reference: https://www.linuxtopia.org/online_books/advanced_bash_scripting_guide/options.html

Reply:
As you can see the script, this script needs to control the exit code by itself. So I didn't use that option.

Reply:
I created another pr to improve this script :) Please review this pr - https://github.com/dapr/dapr/pull/831 even if it was merged already.